lib/base/RequestClient.js

```diff
@@ -107,4 +117,27 @@ RequestClient.prototype.request = function(opts) {
   return deferred.promise;
 };
 
+function parseRequest(options) {
+  console.log('-- BEGIN Twilio API Request --');
+  console.log(`${options.method} ${options.url}`);
+
+  if (options.data) {
+    console.log('Form data:');
+    console.log(options.data);
+  }
+
+  if (options.params && Object.keys(options.params).length > 0) {
```

Comment:
What's the advantage of checking the key length here but not elsewhere?

Reply:
Ops! This was originally left in when this function was parsing the response but it is no longer needed.